docs/sites.yml

```
@@ -3769,3 +3769,16 @@
   built_by_url: "https://www.nearform.com/"
   description: >
     Tools to help diagnose and pinpoint Node.js performance issues.
+- title: Crispin Porter Bogusky
+  url: https://cpbgroup.com/
+  main_url: https://cpbgroup.com/
+  description: >
+    We solve the world’s toughest communications problems with the most quantifiably potent creative assets.
+  categories:
+    - Creative
```

If you're an Agency, I'd replace it with that.

```suggestion
    - Agency
```